Decred Wallet/Custom Views/Button.swift

```diff
@@ -10,24 +10,147 @@ import UIKit
 
 @IBDesignable
 class Button: UIButton {
-    @IBInspectable var borderColor: UIColor = UIColor.clear {
-        didSet {
-            self.layer.borderColor = self.borderColor.cgColor
-            self.setNeedsLayout()
-        }
-    }
-    
-    @IBInspectable var borderWidth: CGFloat = 0 {
-        didSet {
-            self.layer.borderWidth = self.borderWidth
-            self.setNeedsLayout()
-        }
-    }
-    
-    @IBInspectable var cornerRadius: CGFloat = 0 {
-        didSet {
-            self.layer.cornerRadius = self.cornerRadius
-            self.setNeedsLayout()
-        }
-    }
+	private var originalButtonText: String?
+	public var isLoading: Bool = false
+	lazy private var loaderLabel: UILabel = {
+		let loaderLabel = UILabel()
+		loaderLabel.translatesAutoresizingMaskIntoConstraints = false
+		loaderLabel.font = UIFont(name: "SourceSansPro-Regular", size: 18)
+		loaderLabel.textColor = UIColor.appColors.darkBluishGray
+		return loaderLabel
+	}()
+	lazy private var loaderIcon: UIImageView = {
+		let loaderIcon = UIImageView()
+		loaderIcon.translatesAutoresizingMaskIntoConstraints = false
+		loaderIcon.image = UIImage(named: "btn_spinner")
+		return loaderIcon
+	}()
+	lazy private var loaderView: UIView = {
+		let loaderView = UIView()
+		loaderView.isHidden = true
+		self.addSubview(loaderView)
+
+		loaderView.translatesAutoresizingMaskIntoConstraints = false
+		loaderView.centerXAnchor.constraint(equalTo: self.centerXAnchor).isActive = true
+		loaderView.centerYAnchor.constraint(equalTo: self.centerYAnchor).isActive = true
+
+		loaderView.addSubview(loaderIcon)
+		loaderIcon.widthAnchor.constraint(equalToConstant: 20).isActive = true
+		loaderIcon.heightAnchor.constraint(equalToConstant: 20).isActive = true
+		loaderIcon.leadingAnchor.constraint(equalTo: loaderView.leadingAnchor).isActive = true
+		loaderIcon.topAnchor.constraint(equalTo: loaderView.topAnchor).isActive = true
+		loaderIcon.bottomAnchor.constraint(equalTo: loaderView.bottomAnchor).isActive = true
+
+		loaderView.addSubview(loaderLabel)
+		loaderLabel.leadingAnchor.constraint(equalTo: loaderIcon.trailingAnchor, constant: 10).isActive = true
+		loaderLabel.trailingAnchor.constraint(equalTo: loaderView.trailingAnchor).isActive = true
+		loaderLabel.centerYAnchor.constraint(equalTo: loaderIcon.centerYAnchor).isActive = true
+
+		return loaderView
+	}()
+
+	@IBInspectable var loaderTextStringKey: String = "" {
+		didSet {
+			loaderLabel.text = NSLocalizedString(loaderTextStringKey, comment: "")
```

Comment:
loaderLabel.text = LocalizedStrings.loaderTextString

Reply:
`LocalizedStrings.loaderTextString` is not working here. With this function, the localized text of `loaderLabel` can be set from storyboard.

Reply:
`LocalizedStrings.loaderTextString` is not working here. With this function I wanted to add a possibility for the text of `loadLabel` could be set by a localized string key from the storyboard